test/Test/BaseDAO/Proposal.hs

```diff
@@ -396,6 +404,7 @@ flushDecisionLambda = uncapsNettest $ do
         }
   callFrom (AddressResolved owner2) dao (Call @"Vote") [upvote]
 
+  advanceTime (sec 10)
```

Comment:
@gromakovsky I have removed the workaround and use `advanceTime` instead. It seems to work on emulator but in the real network, it seems to not advance the time at all.

Hope to get your advice on whether or not I use the function correctly.

Reply:
Actually the reason was because I set the `voting_period` to short (2 seconds). I've updated this to (20 seconds), and it works fine now.

Hope to get your review on this.